boot.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/deis/riak/chans"
+	"github.com/deis/riak/clustersrv"
+	"github.com/deis/riak/config"
+	"github.com/deis/riak/riak"
+)
+
+func main() {
+	conf, err := config.Get()
+	if err != nil {
+		log.Printf("Error getting config (%s)", err)
+		os.Exit(1)
+	}
+
+	cmdDoneCh := make(chan error)
+	serverDoneCh := make(chan error)
+	if !conf.RiakMaster {
+		// non-bootstrap nodes should start a riak server and join
+		log.Printf("Starting as a bootstrap node")
+		go func() {
+			httpClient := &http.Client{}
+			clusterServerURL := clustersrv.URLFromConfig(conf)
+			if err != nil {
+				cmdDoneCh <- err
+				return
+			}
+
+			if err := riak.Start(); err != nil {
+				cmdDoneCh <- err
+				return
+			}
+
+			if err := riak.Join(httpClient, clusterServerURL); err != nil {
+				cmdDoneCh <- err
+				return
+			}
+		}()
+	} else {
+		// bootstrap nodes should start (not join) a riak server and start the cluster server
+		log.Printf("Starting as a bootstrap node")
```

Comment:
Fixed in https://github.com/arschles/deis-riak/commit/89befd16a7bd80741894d9289a90fb6c68c0d176